tests/framework/microvm.py

```diff
@@ -497,6 +537,32 @@ def start(self):
         response = self.actions.put(action_type='InstanceStart')
         assert self._api_session.is_status_no_content(response.status_code)
 
+    def start_console_logger(self, log_fifo):
+        """
+        Start a thread that monitors the microVM console.
+
+        The console output will be redirected to the log file.
+        """
+        def monitor_fd(microvm, path):
+            try:
+                fd = open(path, "r")
+                while True:
+                    if microvm.logging_thread.stopped():
+                        return
+                    data = fd.readline()
+                    if data:
+                        microvm.append_to_log_data(data)
+            except IOError as error:
+                print(str(error))
```

Comment:
Print statements are not necessarily shown by pytest.
You could use a logger instance declared at the top of the file and output the error there:

```
import logging
log = logging.getLogger("microvm")

...

log.error(f"[{microvm.id}] IOError while monitoring fd: {error}")
```

Reply:
Done.